npm/private/npm_package_store.bzl

```diff
@@ -240,6 +242,16 @@ def _npm_package_store_impl(ctx):
                     ],
                     mnemonic = "NpmPackageExtract",
                     progress_message = "Extracting npm package {}@{}".format(package, version),
+                    # Workaround https://github.com/bazelbuild/bazel-central-registry/issues/2256
+                    # Always override the locale to give better hermeticity.
+                    # See https://github.com/bazelbuild/rules_java/blob/767e4410850453a10ccf89aa1cededf9de05c72e/toolchains/utf8_environment.bzl
+                    env = {
+                        "LC_CTYPE":
```

Comment:
Note @jbedard I repeated the testing to verify that this environment variable also controls the bsdtar behavior. Based on some reading, `LC_ALL` is overkill, this is the surgical value to choose, and also what rules_java is using, see link above.